### DISCUSSION TITLE
How to set release name and release note when using upload_to_play_store?

### DISCUSSION THREAD

**Discussion by savan15** [fastlane/fastlane#21100] [General]
I have using fastlane to deploy my android app on google play store. everything is up to date but i have one question how can i set this two value for google play console.

 As per @vnen new API is that but how can we use it in our FastFile of fastlane.

Also @joshdholtz mentioned that new release that and available but how can we do.

Please help me i have stuck on above issue.

I want to set this 2 fields of google play console.

![Screenshot 2023-03-03 135807](https://user-images.githubusercontent.com/75606567/222683161-6e08e2f4-5dcd-4251-b8fe-d6f6011ad29d.png)

**Comment by slavdok** [comment]
You can do:

`upload_to_play_store(version_name: "string")`
or set ENV variable `SUPPLY_VERSION_NAME`

By default, it will use `versionName` from AndroidManifest.xml
Note, this is probably not the only parameter you need, but that's what controls the version name displayed in Play Console

As for uploading change log, it's automatic unless you disable it with `skip_upload_changelogs: true` parameter.
It's all explained here https://docs.fastlane.tools/actions/upload_to_play_store/ or if you run `fastlane action upload_to_play_store`

**Reply by savan15** [reply]
@slavdok 

Thanks for your reply

but how can we set out "release notes"?

**Reply by slavdok** [reply]
I don't understand your question. It's in the link I provided under "Changelogs (What's new)" section

**Reply by savan15** [reply]
@slavdok 
I am talking about this filed. how can set value from fastlane when upload app to play store.

Like when we upload our app manually in play store it will ask this details to fill so how can we do same in fastlane.

![Screenshot 2023-03-09 103254](https://user-images.githubusercontent.com/75606567/223925375-6c24db0c-1e73-4f78-9267-4bcfe2fbb825.png)

**Reply by slavdok** [reply]
And I've answered several times:
It's explained here https://docs.fastlane.tools/actions/upload_to_play_store/
Under section "Changelogs (What's new)"
![image](https://user-images.githubusercontent.com/54748546/224157414-5567048e-fad2-46f9-8d5a-5d65c08d24d4.png)

Unless you have `skip_upload_changelogs: true` in your `upload_to_play_store`, it will **automatically** look for "release notes" in there.

If you haven't run `fastlane supply init` as per documentation, or want to use a different metadata directory, you can pass `metadata_path: '/path/to/metadata'` parameter to `upload_to_play_store`, or use `SUPPLY_METADATA_PATH` ENV variable

**Reply by savan15** [reply]
Thanks for your quick reply

i will look into it.

**Reply by Vikrant758** [reply]
@slavdok Thanks for the answer.
